packages/slate/src/changes/by-key.js

```diff
@@ -422,8 +422,23 @@ Changes.replaceNodeByKey = (change, key, newNode, options = {}) => {
   const node = document.getNode(key)
   const parent = document.getParent(key)
   const index = parent.nodes.indexOf(node)
+  const { selection } = change.value
```

Comment:
Can you move this line up to the one earlier:

```js
const { document, selection } = value
```

Reply:
Fixed~